HelpCentre/QA/HelpCentre: Intro to HelpCentre.md

```diff
@@ -0,0 +1,1221 @@
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  search_msg = page.body.body.text.value.message
+
+  user_question =
+    ask("""
+    @search_msg
+    """)
+
+  get_latest_msg =
+    get(
+      "https://whatsapp-praekelt-cloud.turn.io/v1/contacts/@contact.whatsapp_id/messages",
+      timeout: 5_000,
+      cache_ttl: 60_000,
+      headers: [
+        [
+          "Authorization",
+          "Bearer @global.settings.turn_qa_token"
+        ],
+        ["Accept", "application/vnd.v1+json"]
+      ]
+    )
+
+  inbound_message_id = get_latest_msg.body.messages[0].id
+  inbound_message_type = get_latest_msg.body.messages[0].type
+  log("External message id = @inbound_message_id")
+  update_contact(route_to_operator_origin: "search_myhealth")
+  update_contact(route_to_operator_search_text: "@user_question")
+end
+
+```
+
+Add markdown here
+
+# Tech Support
+
+```stack
+card TechSupport, then: CheckInbound do
+  inbound_query_type = "tech_support"
+  inbound_query_urgency = "low"
+  num_inbound_attempts = 0
+
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "plat_help_technical_issue_prompt"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  tech_support_msg = page.body.body.text.value.message
+
+  user_question =
+    ask("""
+    @tech_support_msg
+    """)
+
+  get_latest_msg =
+    get(
+      "https://whatsapp-praekelt-cloud.turn.io/v1/contacts/@contact.whatsapp_id/messages",
+      timeout: 5_000,
+      cache_ttl: 60_000,
+      headers: [
+        [
+          "Authorization",
+          "Bearer @global.settings.turn_qa_token"
+        ],
+        ["Accept", "application/vnd.v1+json"]
+      ]
+    )
+
+  inbound_message_id = get_latest_msg.body.messages[0].id
+  inbound_message_type = get_latest_msg.body.messages[0].type
+  log("External message id = @inbound_message_id")
+  log("External message type = @inbound_message_type")
+  update_contact(route_to_operator_origin: "tech_support")
+  update_contact(route_to_operator_search_text: "@user_question")
+end
+
+```
+
+# Call AAQ API and Route Results
+
+```stack
+card CheckInbound when num_inbound_attempts > 1 and helpdesk_open == true, then: SendToHelpdesk do
+  log("Too many failed attempts (@num_inbound_attempts), Helpdesk is open, routing there")
+  update_contact(route_to_operator_origin: "failed_attempts")
+end
+
+card CheckInbound
+     when user_question == "image" or user_question == "audio" or user_question == "video",
```

Comment:
Is this how Turn tells us an input is media in stacks? It seems a bit odd, what happens if you send an image with a caption?

Reply:
Yea something is wrong here.  When I add a caption it doesn't detect it as an image.  I wonder if that's not actually what we want though.  Also added to list of things to check with Lauren

Reply:
Nah I tested some more this morning.  

When you send in an image using the + icon and selecting 'Photos & videos', it comes through as either "video" or "image", which we do have guards for.

When you send in an image using the + icon and selecting 'File', even if you add a caption, the text comes through as "document", which we did not have a guard for. I have now added one

Voicenotes come through as "audio"